sumologic/resource_sumologic_monitors_library_monitor.go

```diff
@@ -135,10 +135,12 @@ func resourceSumologicMonitorsLibraryMonitor() *schema.Resource {
 									"action_type": {
 										Type:     schema.TypeString,
 										Optional: true,
+										Computed: true,
```

Comment:
can we add a deprecation note to this attribute: https://www.terraform.io/docs/extend/best-practices/deprecations.html